base/src/model.rs

```diff
@@ -1172,6 +1172,18 @@ impl Model {
         }))
     }
 
+    /// Returns a boolean representing whether the cell contains a formula or not.
+    pub fn cell_contains_formula(&self, sheet_index: u32, row: i32, column: i32) -> bool {
+        let cell = &self.workbook.worksheets[sheet_index as usize].sheet_data[&row][&column];
```

Comment:
Although I do this quite a lot, we shouldn't.  If the index is wrong this will panic. We should do things like:

```rust
let cell = self.workbook.worksheet(sheet)?.cell(row, column);
```
That will return an error if `(sheet, row, column)` are not valid parameters.
We shouldn't do this in any case, but we should specially avoid it in the public API.
So basically not call to the public API should be able to panic.